src/routes/opo/manage-fleet.js

```diff
@@ -5,7 +5,13 @@ export function get(req, res) {
 
 export function post(req, res) {
   const { name, type } = req.body
-  req.db.run('INSERT INTO vehicles (name, type) VALUES (?, ?)', name, type)
+  const vehicles = req.db.all('SELECT id, name, type FROM vehicles')
+  const vehicleExists = vehicles.some(vehicle => vehicle.name === name)
+  if (vehicleExists) {
+    req.db.run('UPDATE vehicles SET active = 1 WHERE name = ?', [name])
+  } else {
+    req.db.run('INSERT INTO vehicles (name, type) VALUES (?, ?)', name, type)
+  }
```

Comment:
This looks like it works, but can you do it in one query with an [upsert](https://sqlite.org/lang_upsert.html) instead? Speed isn't _that_ important with this app (your solution would be plenty fast) but it does have the advantage of being both faster and declarative.